panel/src/components/View/Buttons/index.js

```diff
@@ -0,0 +1,32 @@
+import Add from "./AddButton.vue";
+import AddLanguages from "./AddLanguagesButton.vue";
+import AddUsers from "./AddUsersButton.vue";
+import Languages from "./LanguagesButton.vue";
+import Preview from "./PreviewButton.vue";
+import Remove from "./RemoveButton.vue";
+import RemoveLanguage from "./RemoveLanguageButton.vue";
+import Settings from "./SettingsButton.vue";
+import Status from "./StatusButton.vue";
+import Theme from "./ThemeButton.vue";
+
+import Buttons from "./Buttons.vue";
+
+export default {
+	install(app) {
+		app.component("k-view-add-button", Add);
+		app.component("k-view-add-languages-button", AddLanguages);
+		app.component("k-view-add-users-button", AddUsers);
```

Comment:
I know that this is useful to match the view name with the right button, but the plural feels off here to me.

Reply:
Do you have a suggestion how to do the matching instead?

Reply:
I made up my mind. Let's keep it like it is. We are overcomplicating this PR and I really think that it is at a point where it is more than clean enough to be in the alpha.